### ISSUE TITLE
Clean Up data view

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#214]
## User story
1. As a User
2. I want a cleaner UI
3. So that the usability improves

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
